pkg/controller/operands/cdi.go

```diff
@@ -81,11 +82,14 @@ func (h *cdiHooks) updateCr(req *common.HcoRequest, Client client.Client, exists
 		return false, false, errors.New("can't convert to CDI")
 	}
 
-	// HCO reconciles the CR for CDI excluding the `spec.CDIConfig`,
```

Comment:
can we please keep this comment until we will properly reconcile the whole spec stanza?